retry.go

```diff
@@ -13,12 +13,14 @@ type Ready func() bool
 // or errors, returning success and error (if any).
 // To stop waiting, send on the stop channel.z
 func Await(ready Ready, maxTries int) bool {
```

Comment:
+1 for renaming to `maxRetries` and updating the doc comment as needed.

Reply:
Sounds good